postcss.config.js

```diff
@@ -34,7 +34,7 @@ const variables = {
 module.exports = {
   plugins: {
     'postcss-easy-import': {},
-    'postcss-cssnext': {},
+    'postcss-cssnext': { warnForDuplicates: false },
```

Comment:
autoprefixer is loaded twice since nested and cssnest uses it.

Btw, we need to move to `postcss-preset-env`